quickstarts/java/jdbc-executebatch/config.yml

```diff
@@ -0,0 +1,38 @@
+name: jdbc-executebatch
+title: JDBC-ExecuteBatch Java Agent Extension
+description: |
+  # What is executeBatch call?
+
+  ExecuteBatch method in jdbc submits a batch of commands to the database for execution and if all commands execute successfully,returns an array of update counts. 
+  The int elements of the array that is returned are ordered to correspond to the commands in the batch, which are ordered according to the order in which they were added to the batch
+    Once deployed the instrumentation will start to track the call to the executeBatch method as a database call. It will show information related to the call in distributed traces.
+  In particular, in the Performance tab for the call it will show the query as "Batch Execute n Queries" where n is the number of queries executed as part of the batch.
+  - For calls to addBatch(String sql), it will represent the number of calls to this method before the executeBatch method is executed. 
+  - For PreparedStatment and CallableStatment it will represent the number of times that addBatch() is called.
+
+summary: |
+  Java agent extension to monitor  the executeBatch method of the Statement, PreparedStatement and CallableStatement interfaces as a database call.
+
+
+icon: icon.svg
+logo: logo.svg
+level: Community
+website: https://docs.oracle.com/javase/7/docs/api/java/sql/Statement.html#executeBatch() 
+
+keywords:
+  - nrlabs
+  - nrlabs-data
+  - apm
+  - java
+  - jdbc
+  - executebatch 
+
+authors:
+  - New Relic Labs
+
+documentation:
+  - name: Java agent extension for JDBC ExecuteBatch method
+    description: |
+      Java Agent instrumentation extension for tracking executeBatch method of the Statement, PreparedStatement and CallableStatement interfaces as a database call.
+    url: >-
+      https://github.com/newrelic-experimental/newrelic-java-JDBC-ExecuteBatch
```

Comment:
This points to an experimental repository, is there actual installation documentation to reference on docs.newrelic.com

Reply:
@jpvajda  these assets live in experimental,  we do not have any references in new relic docs.  we use java agent api and  weaver capability to extended java agent instrumentation capabilities.